src/Nethermind/Nethermind.Evm/Tracing/ITxTracer.cs

```diff
@@ -166,9 +166,10 @@ public interface ITxTracer : IWorldStateTracer, IDisposable
     /// <param name="gas"></param>
     /// <param name="opcode"></param>
     /// <param name="pc"></param>
+    /// <param name="executingAccount"></param>
     /// <param name="isPostMerge"></param>
     /// <remarks>Depends on <see cref="IsTracingInstructions"/></remarks>
-    void StartOperation(int depth, long gas, Instruction opcode, int pc, bool isPostMerge = false);
+    void StartOperation(int depth, long gas, Instruction opcode, int pc, Address executingAccount, bool isPostMerge = false);
```

Comment:
Question to reviewers - do you have any concerns with adding the `executingAccount` information to `StartOperation`? The logic to track the executing account could most likely could be done without this extra param, however I feel like this approach is much cleaner and minimizes repetitive code in the VM and the tracer

Reply:
I think you can easily extract executing account in `ReportAction`. I would prefer not to change any of the method parameters if we can avoid it.